src/layout/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useState } from 'react';
+
+import { Tabs as DesignsystemetTabs } from '@digdir/designsystemet-react';
+
+import { useRegisterNodeNavigationHandler } from 'src/features/form/layout/NavigateToNode';
+import { useLanguage } from 'src/features/language/useLanguage';
+import { GenericComponent } from 'src/layout/GenericComponent';
+import type { PropsFromGenericComponent } from 'src/layout';
+
+export const Tabs = ({ node }: PropsFromGenericComponent<'Tabs'>) => {
+  const [activeTab, setActiveTab] = useState<string | undefined>(
+    node.item.defaultTab ?? node.item.tabsInternal.at(0)?.id,
+  );
+
+  useRegisterNodeNavigationHandler((targetNode) => {
+    for (const parent of targetNode.parents() ?? []) {
+      const tabsInternal = parent.item['tabsInternal'];
+      const targetTabId = tabsInternal?.find((tab) =>
+        tab.childNodes.some((child) => child.item.id === targetNode.item.id),
+      )?.id;
+      if (targetTabId) {
+        setActiveTab(targetTabId);
+        return true;
+      }
+    }
+    return false;
+  });
+
+  const tabs = node.item.tabsInternal;
+  return (
+    <DesignsystemetTabs
+      defaultValue={activeTab}
+      value={activeTab}
+      onChange={(tabId) => setActiveTab(tabId)}
+      size={node.item.size}
+    >
+      <DesignsystemetTabs.List>
+        {tabs.map((tab) => (
+          <TabHeader
+            key={tab.id}
+            id={tab.id}
+            title={tab.title}
+            icon={tab.icon}
+            isActive={tab.id === activeTab}
+          />
+        ))}
+      </DesignsystemetTabs.List>
+      {tabs.map((tab) => (
+        <DesignsystemetTabs.Content
+          key={tab.id}
+          value={tab.id}
+          role='tabpanel'
+          style={{
+            backgroundColor: 'white',
```

Comment:
Do we have to have this? I'm curious how Tabs would look inside for example a Panel or a RepeatingGroup, which have other background colors than a plain top-level form component.